src/icons/navigator/Swap.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react'
+import Svg, { Path } from 'react-native-svg'
+
+export function Swap() {
+  return (
+    <Svg width={32} height={32} viewBox="0 0 32 32" fill="none">
```

Comment:
Would be nice to have a testID.
`<Svg testID="swap" width={32} height={32} viewBox="0 0 32 32" fill="none">`

Reply:
I just did that. Thanks!